factorial.rb

```diff
@@ -0,0 +1,11 @@
+#factorial de un  numero 
+puts "numero : "
+v2= gets.chomp.to_i
```

Comment:
try to avoid naming variables with acronyms, if another developer reads your code he might have problems to know what the variable will be used for. (teamwork)